pkg/infoschema/cache.go

```diff
@@ -59,6 +60,13 @@ func NewCache(r autoid.Requirement, capacity int) *InfoCache {
 	}
 }
 
+// GetAndResetRecentInfoSchemaTS provides the min start ts for infosync.InfoSyncer.
+func (h *InfoCache) GetAndResetRecentInfoSchemaTS(now uint64) uint64 {
+	ret := atomic.LoadUint64(&h.Data.recentMinTS)
+	atomic.StoreUint64(&h.Data.recentMinTS, now)
```

Comment:
Can we use `atomic.Uint64` instead?

Reply:
Is it possible that an `infoschemaV2` is acccessed in a lower frequence than `ReportMinStartTS`, so that a tick of `ReportMinStartTS` may miss this `infoschemaV2` instance, which allows GC to proceed, the `infoschemaV2` instance becomes unavailable again?

Reply:
In theory, that is possible, so there is still potential risk. I have discussed that with @lcwangchao 
But in practice I think this fix is good enough for the most common cases. @MyonKeminta

Reply:
Ok. But I suggest this details can be noted in the comments in the code.
Rest LGTM.

Reply:
Done